sunkit_pyvista/tests/test_pyvista.py

```diff
@@ -131,7 +134,7 @@ def test_multi_block(plotter):
     assert plotter.all_meshes['solar_axis'][0].n_points == 101
 
 
-def test_field_lines(aia171_test_map, plotter):
+def test_field_lines_and_color_func(aia171_test_map, plotter):
```

Comment:
Is this a figure test? If not can we add a figure test with the custom color function?

Reply:
I was going to suggest that, do we have a new figure test for this one? Or modify the existing figure to use colorfunc instead of the default colours

Reply:
Lets modify the existing one

Reply:
Yep, I've done that